charts/merlin/templates/_config.tpl

```diff
@@ -19,39 +19,39 @@ ImageBuilderConfig:
   BaseImages:
     3.7.*:
       ImageName: ghcr.io/caraml-dev/merlin/merlin-pyfunc-base-py37:{{ printf "%s" $tag }}
-      DockerfilePath: "docker/Dockerfile"
+      DockerfilePath: "python/pyfunc-server/docker/Dockerfile"
       BuildContextURI: "git://github.com/caraml-dev/merlin.git#{{ printf "%s" $reference }}"
     3.8.*:
       ImageName: ghcr.io/caraml-dev/merlin/merlin-pyfunc-base-py38:{{ printf "%s" $tag }}
-      DockerfilePath: "docker/Dockerfile"
+      DockerfilePath: "python/pyfunc-server/docker/Dockerfile"
       BuildContextURI: "git://github.com/caraml-dev/merlin.git#{{ printf "%s" $reference }}"
     3.9.*:
       ImageName: ghcr.io/caraml-dev/merlin/merlin-pyfunc-base-py39:{{ printf "%s" $tag }}
-      DockerfilePath: "docker/Dockerfile"
+      DockerfilePath: "python/pyfunc-server/docker/Dockerfile"
       BuildContextURI: "git://github.com/caraml-dev/merlin.git#{{ printf "%s" $reference }}"
     3.10.*:
       ImageName: ghcr.io/caraml-dev/merlin/merlin-pyfunc-base-py310:{{ printf "%s" $tag }}
-      DockerfilePath: "docker/Dockerfile"
+      DockerfilePath: "python/pyfunc-server/docker/Dockerfile"
       BuildContextURI: "git://github.com/caraml-dev/merlin.git#{{ printf "%s" $reference }}"
   PredictionJobBaseImages:
     3.7.*:
       ImageName: ghcr.io/caraml-dev/merlin/merlin-pyspark-base-py37:{{ printf "%s" $tag }}
-      DockerfilePath: "docker/app.Dockerfile"
+      DockerfilePath: "python/pyfunc-server/docker/Dockerfile"
```

Comment:
the path is different for jobs:
```
          DockerfilePath: "python/batch-predictor/docker/app.Dockerfile"
```

Reply:
https://github.com/caraml-dev/merlin/blob/main/python/batch-predictor/docker/app.Dockerfile

Reply:
the Dockerfile is still the same, shouldn't it be app.Dockerfile?

Reply:
damn, fixed.